client/src/components/buttons/DeleteButton.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import colors from '../../utils/colors';
+import Delete from '../../assets/deleteButton.svg';
+import { CoinContext } from '../../utils/CoinContext';
+import PropTypes from 'prop-types';
+
+const DeleteImage = styled.img`
+  width: 100%;
+  height: 100%;
+`;
```

Comment:
If you take a look at these lines only, how would you describe this image?
I would say, it's an image with full width and height. 
You named it `DeleteImage` because you are using this image inside a button that deletes something.
But if you think in a more generic way, you shouldn't limit the usage based on the naming.
I would call it `FullImage` or `StrechImage`.

This makes it easier to reuse and outsource these components.